docs/guides/installation-helm.md

```diff
@@ -0,0 +1,78 @@
+This chart installs the [TriggerMesh Cloud Native Integration Platform](https://github.com/triggermesh/triggermesh) on a Kubernetes cluster.
+
+!!! note
+    This is an alternative method of installation. A straight forward `kubectl apply` is actually all [it takes](installation.md).
+
+## TL;DR;
+
+If you are familiar with [Helm](https://helm.sh/) just go for it:
+
+```console
+$ helm repo add triggermesh https://storage.googleapis.com/triggermesh-charts
+$ helm install -n triggermesh triggermesh triggermesh/triggermesh --create-namespace
+```
+
+## Prerequisites
+  - Kubernetes 1.20+
+  - Helm 3.0+
+  - Knative v0.25+
+
+## Installing the Chart
+
+Add the TriggerMesh chart repository to Helm:
+
+```console
+$ helm repo add triggermesh https://storage.googleapis.com/triggermesh-charts
+```
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release triggermesh/triggermesh
+```
+
+!!! info
+    If you face any issues please let us know about it and create an [issue](https://github.com/triggermesh/triggermesh/issues/new)
```

Comment:
That's not the chart's repo though, is it expected?

Reply:
yes, because the chart repo is still private